crates/storage/src/lazy/mapping.rs

```diff
@@ -85,14 +85,30 @@ where
     where
         Q: scale::EncodeLike<K>,
     {
-        pull_packed_root_opt(&self.storage_key(key))
+        pull_packed_root_opt(&self.storage_key(&key))
+    }
+
+    /// Clears the value at `key` from storage.
+    pub fn clear_entry<Q>(&self, key: Q)
```

Comment:
```suggestion
    pub fn remove<Q>(&self, key: Q)
```
I would name it `remove`, since the term entry` doesn't appear anywhere else, there's no `get_entry`, no `insert_entry`. Also it can be misleading, since we also have an entry API for some data structures.

You'll have to adapt the other occurrences as well.

Reply:
I too think that `remove` as a name is better.

Reply:
I want to reserve `remove` in case we want to add an analogue to `HashMap/BTreeMap::remove`. Although I guess we can also use `take` for that. What do you guys think?

Reply:
I agree. I'm wondering, do you think you'll get away without `take` for the examples?

Reply:
So far I haven't needed it, but I've only gone through `ERC20`, `ERC1155`, and just started `ERC721`. Either way, its behaviour can be replicated by doing `get()` followed by `clear_entry` (or whatever we name it)